client/src/helpers/fhir/patientHelper.ts

```diff
@@ -8,7 +8,7 @@ export function getRAMQValue(patient: Patient): string | undefined {
 export function formatRamq(value: string): string {
   const newValue = value.toUpperCase();
   return newValue
-    .replaceAll(/\s/g, '')
+    .replace(/\s/g, '')
```

Comment:
Node doesn't support `replaceAll` but the `g` flag on the regex does the same thing